### PR TITLE
Added markdown support in chat

### PR DESCRIPTION
**Pull request by scoheleach**

### CODE REVIEW

File: src/main/webapp/css/main.css [hunk 1]
```diff
@@ -56,3 +56,11 @@ input {
 button {
   font-size: 18px;
 }
+
+#previewBox {
```
Comment (scoheleach) [inline src/main/webapp/css/main.css:60]: this is from when i was going to add a preview box. i'm still interested in implementing that later, maybe for the open project, but it isn't totally relevant to the functionality of this pr

Comment (ghost) [reply src/main/webapp/css/main.css:60]: that's a great idea!

**Review by ghost — APPROVED** [review]
Good job

**Review by jwang281 — APPROVED** [review]
Great work!

**Pull request merged**